README.rst

```diff
@@ -81,7 +81,7 @@ SSL connections that are self-signed (ex: Heroku):
         "BACKEND": "channels_redis.pubsub.RedisPubSubChannelLayer",
         "CONFIG": {
             "hosts":[{
-                "address": "rediss://user@host:port",  # "REDIS_TLS_URL"
```

Comment:
I'm not sure this one is a typo. `rediss://` looks to be a valid scheme name for TLS

https://github.com/redis/node-redis/blob/master/docs/client-configuration.md
https://www.iana.org/assignments/uri-schemes/prov/rediss

Reply:
Didn't know that. Thank you for the information!